### PR TITLE
Fix occasional ES 503 test errors

### PR DESCRIPTION
**Pull request by mparent61**
Caused by accessing ES indices before shard allocation completed. Seems most apparent in test cases that don't call a "refresh" on search objects, like token auth tests. 

Reference: http://stackoverflow.com/a/30311714

Sample Error: https://travis-ci.org/theonion/django-bulbs/jobs/115362885

Was able to recreate + verify fix locally. Travis builds seem fixed too. 

Also threw in some lint fixes (see comments).

### CODE REVIEW

**Comment by mparent61** [comment]
Also applies some lint + PY2/PY3 best practices

File: bulbs/utils/test/__init__.py [hunk 1]
```diff
@@ -35,7 +34,7 @@ def make_content(*args, **kwargs):
         models = indexable_registry.families[Content]
         model_keys = []
         for key in models.keys():
-            if not key in ['content_content', 'poll_poll']:
+            if key not in ['content_content', 'poll_poll']:
```
Comment (mparent61) [inline bulbs/utils/test/__init__.py]: Lint

File: bulbs/utils/test/__init__.py [hunk 2]
```diff
@@ -54,7 +53,7 @@ def default(self, value):
         iso = _iso_datetime(value)
         if iso:
             return iso
-        if not PY3 and isinstance(value, str):
+        if PY2 and isinstance(value, str):
```
Comment (mparent61) [inline bulbs/utils/test/__init__.py]: PY3+ best practice, better to check PY2, so we don't break on future PY4 (which will be more similar to 3 than 2)

**Comment by mparent61** [comment]
@benghaziboy @MichaelButkovic @collin

**Comment by MichaelButkovic** [comment]
:+1: very nice

**Pull request merged**